src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3830,13 +3830,17 @@ components:
       type: object
       required:
         - non_fungible_id
-        - last_updated_at_state_version
         - is_burned
+        - last_updated_at_state_version
       properties:
         non_fungible_id:
           $ref: "#/components/schemas/NonFungibleId"
         owning_vault_address:
```

Comment:
can we add short description what is that? (i know you haven't touched it)

Reply:
`last_updated_at_state_version` references `LastUpdatedAtStateVersion` that defines the appropriate description already.